src/states/bootstrapping_peer.rs

```diff
@@ -465,11 +465,11 @@ mod tests {
         let mut sel = mpmc::Select::new();
         machine.register(&mut sel);
 
-        // Blocking step for the first one.
-        let op_index = sel.ready();
+        // Step for the first one.
+        let op_index = unwrap!(sel.try_ready());
```

Comment:
Why do we change that? does it make the name of the function wrong?

Reply:
`ready` is blocking and I had this test hang due to error before. I think it's better to panic than to block.

Reply:
Make sense.